src/algorithms/channel/libs/channel_fsm.h

```diff
@@ -58,6 +58,8 @@ class ChannelFsm
     virtual bool Event_failed_acquisition_repeat();
     virtual bool Event_failed_acquisition_no_repeat();
 
+    inline uint32_t state() { return state_; }
```

Comment:
Same here: `inline uint32_t state() const { return state_; }`

Reply:
Done in commit https://github.com/gnss-sdr/gnss-sdr/pull/419/commits/585754a665345c9629dd327d1d81dfc568cdf722

Reply:
Fixed in commit https://github.com/gnss-sdr/gnss-sdr/pull/419/commits/585754a665345c9629dd327d1d81dfc568cdf722